frontend/packages/topology/src/components/graph-view/components/nodes/BaseNode.tsx

```diff
@@ -53,6 +54,8 @@ type BaseNodeProps = {
   createConnectorAccessVerb?: K8sVerb;
   nodeStatus?: NodeStatus;
   showStatusBackground?: boolean;
+  showAlertStatus?: boolean;
+  alertVariant?: string;
```

Comment:
Instead of new props we should reuse the `nodeStatus` which is already used for other alerts see `WorkloadNode` `nodeStatus={getAggregateStatus(...`